gammapy/modeling/models/temporal.py

```diff
@@ -493,7 +493,7 @@ class LightCurveTemplateTemporalModel(TemporalModel):
     _t_ref_default = Time("2000-01-01")
     t_ref = Parameter("t_ref", _t_ref_default.mjd, unit="day", frozen=True)
 
-    def __init__(self, map, t_ref=None, filename=None):
+    def __init__(self, map, t_ref=None, filename=None, method=None, values_scale=None):
```

Comment:
`method` and `values_scale` can be added to the doctoring

Reply:
I've added a note about them in the docstring.